Game/start_menu.gd

```diff
@@ -2,24 +2,34 @@ extends Control
 
 @export var level_scene: PackedScene
 
+@onready var upload_image_button = $VBoxContainer/HBoxContainer/UploadImageButton
```

Comment:
```suggestion
@onready var upload_image_button = $VBoxContainer/HBoxContainer/UploadImageButton
@onready var load_level_button = $VBoxContainer/HBoxContainer/LoadLevelButton
```
Should include the Load Level button here as well for applying the Dark Mode filter.